project/ScalafixBuild.scala

```diff
@@ -283,10 +280,8 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
   )
 
   private val PreviousScalaVersion = Map(
-    "2.11.12" -> "2.11.11",
-    "2.12.4" -> "2.12.3",
-    "2.12.8" -> "2.12.7",
-    "2.12.10" -> "2.12.8"
+    "2.13.3" -> "2.13.2",
+    "2.12.12" -> "2.12.11"
```

Comment:
ready for the next bumps - unused at the moment but I wanted to keep a list to test and document how it's meant to be used